Makefile

```diff
@@ -97,12 +97,20 @@ vet:
 			echo "and fix them if necessary before submitting the code for reviewal."; \
 		fi
 
+ifeq ($(OS),Windows_NT)
```

Comment:
After much Googling, this heuristic seems as good as any, and has the advantage that its intent is rather obvious (well the `_NT` bit might be misleading, as this same value is rumoured to be used in all versions of Windows since 2000). Other methods look for other obscure env vars and/or rely on trying to execute a command that fails, so I picked this imperfect test as the least bad of a bad bunch.